holodeck/gps/gp_utils.py

```diff
@@ -668,7 +668,11 @@ def hc_from_gp(gp_george, gp_list, gp_george_variance, gp_list_variance,
         mean_pred, mean_pred_unc = gp_list[ii].predict(gp_george[ii].y, [env_pars])
         std_pred, std_pred_unc = gp_list_variance[ii].predict(gp_george_variance[ii].y, [env_pars])
 
-        total_pred_unc = np.sqrt(std_pred**2 + std_pred_unc**2 + mean_pred_unc**2)
+        if include_gp_unc:
+            total_pred_unc = np.sqrt(std_pred**2 + std_pred_unc**2 + mean_pred_unc**2)
```

Comment:
Are these two different cases enough (i.e. `std_pred` vs. `std_pred + std_pred_unc + mean_pred_unc`), or would other permutations be useful?  I'm assuming the idea is to capture the most and least possible varance --- which seems like a good approach!

Reply:
Yes, that was my idea; "True" for how most people would use the `hc_from_gp`, where the "machine uncertainty" of the GPs is included, and "False" for how someone investigating the quality of a stdev GP's predictions. We could also make the new flag something like `unc_weight` and make it a weight vector that multiplies `[std_pred**2,std_pred_unc**2,mean_pred_unc**2]`, so the current "True" would correspond to `[1,1,1]` and the current "False" to `[1,0,0]`. I can't see a clear use case for that at the moment, but it seems like it could potentially be helpful for more involved statistics.

Also, good catch that I should be adding this to `dev` and not `main`!